tee-worker/app-libs/stf/src/trusted_call_result.rs

```diff
@@ -98,4 +98,8 @@ pub struct RequestVCResult {
 	pub vc_index: H256,
 	pub vc_hash: H256,
 	pub vc_payload: AesOutput,
+	// see comments in `lc-vc-task-receiver` why it's prefixed with `pre...`
+	// they should be referenced/used only when the client's local IDGraph is empty
+	pub pre_mutated_id_graph: AesOutput,
```

Comment:
Based on the comments, Shouldn;t we use `Option` here?

Reply:
It's actually a taste of semantics - currently if nothing is mutated then the `mutated_id_graph` will be empty vector, which is fine as well IMO. `Option` will do the job too - if it's more beneficial we can introduce another PR (as all need other `xxxResult` need to be changed too)

Tagging @jonalvarezz here

Reply:
> if nothing is mutated then the mutated_id_graph will be empty vector, which is fine as well IMO

It is perfect for clients as they can skip building the `Option`

Reply:
The trade-off is you always have to perform the decryption :)
A `None` is detected at the first sight

Reply:
Anyways if our client representative Jonathan doesn't have objection I'll leave it as it is :D